### PR TITLE
Add README.md header

### PR DESCRIPTION
**Pull request by qdequele**

### CODE REVIEW

**Comment by curquiza** [comment]
@qdequele could you rebase, please? 🙂

File: README.md [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
-[![Actions Status](https://github.com/meilisearch/meilisearch-php/workflows/Tests/badge.svg)](https://github.com/meilisearch/meilisearch-php/actions)
-[![Latest Stable Version](https://poser.pugx.org/meilisearch/meilisearch-laravel-scout/version)](https://packagist.org/packages/meilisearch/meilisearch-laravel-scout)
-
-The Laravel scout package for MeiliSearch.
-
-MeiliSearch provides an ultra relevant and instant full-text search. Our solution is open-source and you can check out [our repository here](https://github.com/meilisearch/MeiliSearch).
-
-Here is the [MeiliSearch documentation](https://docs.meilisearch.com/) 📖
+<p align="center">
+  <img src="https://res.cloudinary.com/meilisearch/image/upload/v1587402338/SDKs/meilisearch_laravel.svg" alt="MeiliSearch-Laravel Scout" width="200" height="200" />
```
Comment (curquiza) [inline README.md]: Is that logo the real logo of [Laravel Scout](https://github.com/laravel/scout)?

**Review by curquiza — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (qdequele) [reply README.md]: Sorry, I don't know that Laravel Scout has a different logo than Laravel :/

Comment (curquiza) [reply README.md]: Np 🙂

Comment (qdequele) [reply README.md]: Done 👍

**Review by curquiza — APPROVED** [review]
Awesome!!!

**Pull request merged**